src/Raven.Client/Documents/Commands/Batches/BatchCommand.cs

```diff
@@ -159,5 +159,7 @@ public void Dispose()
             foreach (var command in _commands)
                 command?.Dispose();
         }
+
+        public string RaftUniqueRequestId { get; } = Guid.NewGuid().ToString();
```

Comment:
Since we are using that all over the place and there are some magic string here e.g. `string.Empty`. I would say that we need to create internal static class called `RaftIdentifier` with two methods:
1) `NewId` which will return this
2) `OtherNameThatExplainsWhatItDoes` which will return `string.Empty`

Reply:
This will also help centralize the identifiers.